### ISSUE TITLE
ACC-264.1, ACC-270.3, ACC-270.13 (Level A)_ Lists_ Count is not read, user cannot navigate and immediate focus doesnot shift

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3248]
v1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Lists
Affected Check Point:	ACC-264.1,ACC-270.3,ACC-270.13
Short text:	ACC-264.1, ACC-270.3, ACC-270.13 (Level A)_ Lists_ Count is not read, user cannot navigate and immediate focus doesnot shift

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Go to KubeconfigURL:https://kyma-env-broker.cp.stage.kyma.cloud.sap/kubeconfig/DFEFE4E0-E202-4455-8A61-37511B4D7553
•	Click on  the link https://dashboard.stage.kyma.cloud.sap/clusters and select Connect.
•	Upload the kubeconfing file downloaded in the first step. You can either drag and drop it or select the upload icon to choose it from the device.
•	Next Step
•	Connect to cluster, A new screen get opens
•	Open ‘Profile account select ‘Preference’ button 
•	

3. Issue Type: Manual
Observed Behavior:
On accessing the List items in the left navigation region within the dialog ‘Preference’ ..
1.Count of the List items are not being read out.
2. User cannot navigate using the Arrow keys.
3. On selecting the list items the immediate focus doesn’t shifts to the related UI.

Screenshot:
 
Expected Behavior:
It should always be possible to navigate within the control elements using navigation keys.
Here navigation within the menu items should be possible with arrow keys instead of tab keys.Initial focus should be positioned in such a way that it supports to completed all the associated task.

Providing initial focus on first elements of dialog as 'Automatic synchronization’ radio button makes an application more feasible for all screen reader and keyboard users with visual difficulties.

[ACC-264.1, ACC-270.3, ACC-270.13 (Level A)_ Lists_ Count is not read, user cannot navigate and immediate focus doesnot shift.docx](https://github.com/user-attachments/files/16687787/ACC-264.1.ACC-270.3.ACC-270.13.Level.A._.Lists_.Count.is.not.read.user.cannot.navigate.and.immediate.focus.doesnot.shift.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16687789/Issue.processing-09.2021.2.docx)
